PhysIO/code/tapas_physio_main_create_regressors.m

```diff
@@ -53,18 +53,19 @@
 %% 0. Set Default parameters
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-% Remove SPM subfolders from path to avoid conflicts with Matlab in-built
-% functions used in PhysIO
-doCorrectPath = true;
-tapas_physio_check_spm(doCorrectPath);
-% Silently check proper Batch Editor integration, if SPM exists,
-% but don't complain, if there is no SPM installed
-isVerbose = false;
-tapas_physio_check_spm_batch_editor_integration(isVerbose);
+oldPath = path();
 
-% Include subfolders of code to path as well
 pathThis = fileparts(mfilename('fullpath'));
-addpath(genpath(pathThis)); 
+ft_paths = which('ft_defaults', '-all')
```

Comment:
If `rmpath` are moved into separate function, it would be difficult to restore path afterwards.
`tapas_physio_check_spm`  should then return the old path, which will restored at the end.
Will make code more difficult to read.

I think it would be better to run `check_spm` in init, show warnings etc...
And then, in PhysIO, just silently remove and restore fieldtrip. 
If all goes normally, all paths manipulations will be irrelevant for user -- the only exception, that
not catched by `try` is Cntr-C (which, I hope will be used rarely).

Reply:
I see what you mean, but I will have to think about it a bit more what the expected behavior should be. 

While being a very useful function, `tapas_physio_init` is not used by everyone, that's why the incompatible states of the path occur in the first place. But I agree, manipulating paths when calling `main`, but hidden in other functions might also not be best practice...

Reply:
I will propose a compromise:
The path removal in separate function (e.g. `tapas_physio_check_ft`), with a warning stating of
path removal, so user is aware of it, and can restore it if he need.

Similar to:
```
Found installation of FieldTrip at:
'/home/abc/matlab/FieldTrip'
Will remove it from paths for compatibility reason, see
https://github.com/translationalneuromodeling/tapas/issues/166

If needed you can restore it by running:
addpath('/home/abc/matlab/FieldTrip');
ft_defaults;
```

May be warning could be shortened following verbosity setting.

Reply:
That sounds like a good compromise, thank you, Nikita!

I will need some time to integrate it in the internal development branch of PhysIO, but please ping me if nothing happens in the next two weeks.

All the best,
Lars

Reply:
Checks moved to `tapas_physio_check_fieldtrip.m`, so this request should be resolved.